code/FileAttachmentFieldTrack.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/**
+ * Track files as they're uploaded and remove when they've been saved.
+ *
+ * @package  unclecheese/silverstripe-dropzone
+ */
+class FileAttachmentFieldTrack extends DataObject {
+    private static $db = array(
+        'SessionID' => 'Varchar(255)',
+        'ControllerClass' => 'Varchar(60)',
+        'RecordID' => 'Int',
+        'RecordClass' => 'Varchar(60)',
+    );
```

Comment:
It seems like `ControllerClass` , `RecordID` and `RecordClass` are just used for reference purposes, and they're not revealed until the task is run. Maybe there's a case for a ModelAdmin interface here, so the user can actually make some inferences on these values? Otherwise, by the time I know what they are, the record has already been deleted, as far as I can tell.